plugins/resources/replication/librepl.cpp

```diff
@@ -1641,6 +1641,9 @@ class repl_resource : public irods::resource {
                         }
                         else {
                             properties_.set< size_t >( NUM_REPL_KW, static_cast< size_t >( num_repl ) );
+                            irods::log(LOG_NOTICE,
```

Comment:
LOG_WARNING ?    are our other deprecation messages LOG_NOTICE?

Reply:
unixfilesystem resource is the only one which logs deprecation messages and it is using LOG_NOTICE:

https://github.com/irods/irods/blob/d91ec25e619e159c5b9416c7fed2408d31dbf833/plugins/resources/unixfilesystem/libunixfilesystem.cpp#L464-L467

I was also wondering if the message should be logged just here (standing up the resource plugins), or also at the time that the policy is applied to the child list (each time a create is performed in this resource).

Reply:
It may be worth the format call to specify which resource this is, too.

Reply:
definitely worth the time to format a helpful message.   and if production systems are set to WARNING, we still want them to hear about this.

at time of use is fine rather than creation - it will already be in their logs as soon as they use it.  and we don't parse things for meaning at creation/edit of the context string.